app/middleware/validation.middleware.ts

```diff
@@ -1,10 +1,11 @@
 import { plainToClass } from 'class-transformer';
-import { validate, ValidationError } from 'class-validator';
+import { validate } from 'class-validator';
 import * as express from 'express';
 import HttpException from '../exceptions/HttpException';
+import { NextFunction, Request, Response } from 'express';
 
 function bodyValidationMiddleware<T>(type: any): express.RequestHandler {
-  return (req, res, next) => {
+  return (req: Request, res: Response, next: NextFunction) => {
```

Comment:
You should need these types because `bodyValidationMiddleware ` defines it's return type 🤔